packages/table/src/table.tsx

```diff
@@ -489,15 +489,34 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         );
     }
 
-    public resizeRowsByTallestCell(columnIndex: number) {
+    /**
+     * Resize rows based on a given set of column indices.
+     * If no indices are provided, default to using the columns that are currently in the viewport.
+     */
+    public resizeRowsByTallestCells(columnIndices?: number[]) {
```

Comment:
i am inclined to agree with @cmslewis above. the plural felt weird to me to begin with, and now it's clear that it's not even semantically correct.